packages/api/src/event/medical/canvas-event-listener.ts

```diff
@@ -0,0 +1,51 @@
+import CanvasSDK from "@metriport/core/external/canvas/index";
+import { createFullNote } from "@metriport/core/external/canvas/note";
+import { getEnvVarOrFail } from "@metriport/shared";
+import { PatientEvents, patientEvents, PatientEvent } from "./patient-event";
+import { getPatientOrFail } from "../../command/medical/patient/get-patient";
+import { Util } from "../../shared/util";
+
+const log = Util.log(`[CANVAS EVENT LISTENER]`);
+
+export default function () {
+  log(`Setting up listener`);
+  patientEvents().on(PatientEvents.CANVAS_INTEGRATION, async (event: PatientEvent) => {
+    log(`Received event: ${JSON.stringify(event, null, 2)}`);
+    const patient = await getPatientOrFail({ id: event.id, cxId: event.cxId });
+    const patientFirstName = patient.data.firstName;
+
+    let patientA = false;
+    let patientB = false;
+    if (patientFirstName.toLowerCase() === "jane") {
+      patientA = true;
+    } else if (patientFirstName.toLowerCase() === "john") {
```

Comment:
if patient name is jane, we provide one default fake record. If john, we provider another. If any other name we provide no record